src/libraries/Microsoft.Extensions.DependencyInjection/Directory.Build.props

```diff
@@ -1,7 +1,8 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <PackageVersion>5.0.1</PackageVersion>
+    <ServicingVersion>1</ServicingVersion>
+    <!-- This should be pinned to 5.0.0.1 going forward as it is part of aspnetcore ref pack.-->
```

Comment:
Will we still increment the version for .NETFramework?

Reply:
I'm not sure this is fixed.  It looks to me like we will not rev the version on .NETFramework due to the condition in props targets file